docs/tutorial/working-with-frames-and-series/index.md

```diff
@@ -96,6 +98,7 @@ patient_id        | row_id            | date              | dmd_code
 ```
 
 Similarly, the `medications.date` column has many rows per patient.
+Notice that the column is indexed by `patient_id` and `row_id`.
```

Comment:
No: it's only indexed by `row_id`.

Reply:
The primary key is `row_id`. The foreign key is `patient_id`. But if we want to look up the medications for a patient, and we don't want to describe why our data structure of one series plus an index is actually a data structure of two series plus an index, it helps to think of an _index_ as a look-up table.